docs/html/user_guide.rst

```diff
@@ -380,17 +380,17 @@ variable ``PIP_CONFIG_FILE``.
   :file:`C:\\Documents and Settings\\All Users\\Application Data\\pip\\pip.ini`
 * On Windows 7 and later the file is hidden, but writeable at
   :file:`C:\\ProgramData\\pip\\pip.ini`
-* Site-wide configuration is not supported on Windows Vista
+* System-wide configuration is not supported on Windows Vista
```

Comment:
I can see why you used this term (system-wide), and it makes definite sense as a term, but maybe since the PR is meant to get a bit more consistency around terminology it's worth sticking to using "global" here rather than introducing a synonym for it (system-wide), and again having two terms for the same concept?

I.e. `* Global configuration is not supported on` and below similarly?

Reply:
@Julian Oh I understand! lemme change it.